Integrations/Cymulate/Cymulate.yml

```diff
@@ -0,0 +1,67 @@
+category: Vulnerability Management
+commonfields:
+  id: Cymulate
+  version: -1
+configuration:
+- defaultvalue: 179743a665726a99fdee3bb0d10ba3fe
+  display: x-token
+  name: x-token
+  required: true
+  type: 0
+- display: Fetch incidents
+  name: isFetch
+  required: false
+  type: 8
+- display: Incident type
+  name: incidentType
+  required: false
+  type: 13
+- display: Show only 'penatrated' incidents
+  hidden: false
+  name: onlyPenatrated
+  required: false
+  type: 8
+description: Multi-Vector Cyber Attack | Breach and Attack Simulation
+display: Cymulate
+name: Cymulate
+script:
+  commands:
+  - arguments:
+    - default: false
+      description: The incident unique identifier
+      isArray: false
+      name: incident_id
+      required: true
+      secret: false
+    - default: false
+      description: The Cymulate module_type
```

Comment:
are there predefined types? if there are then we have single select input

Reply:
no, currently only static predefine type

Reply:
what do you mean by static predefine type?
I mean it seems that there are predefine list of possible values that module_type can be:
```
	class CymulateModuleTypeEnum(enum.Enum):
    """
       Enum class of module types of cymulate attacks
    """

    APT = '1',
    IMMEDIATE_THREATS = '2',
    HOPPER = '3',
    PHISHING = '4',
    WAF = '5',
    DLP = '6',
    BROWSING = '7',
    EDR = '8',
    MITRE = '9',
    MAIL = '10'
```